### PR TITLE
Optional Validation

### PR DESCRIPTION
**Pull request by tannerlinsley**

### CODE REVIEW

**Comment by tannerlinsley** [comment]
Technically, this just allows a field validator to return no errors if it happens to be undefined

File: src/validation.test.js [hunk 1]
```diff
@@ -31,6 +32,14 @@ test('required', (t) => {
   t.is(new Validation().contains('hi').run('field', '').length, 0)
 })
 
+test.only('optional', (t) => {
```
Comment (jsonmaur) [inline src/validation.test.js]: Take off `.only`

Comment (tannerlinsley) [inline src/validation.test.js]: hahahaha, ooops

**Comment by jsonmaur** [comment]
Awesome. I'll release this when I get those other issues sorted out too.

**Pull request merged**